cpu/stm32/include/periph/cpu_spi.h

```diff
@@ -81,6 +81,36 @@ typedef uint32_t spi_cs_t;
 #define PERIPH_SPI_NEEDS_TRANSFER_REGS
 /** @} */
 
+/* unify names across STM32 families */
+#ifdef SPI_CR1_CPHA_Msk
+#  define STM32_SPI_CPHA_Msk            SPI_CR1_CPHA_Msk
+#endif
+#ifdef SPI_CFG2_CPHA_Msk
+#  define STM32_SPI_CPHA_Msk            SPI_CFG2_CPHA_Msk
+#endif
+#ifdef SPI_CR1_CPOL_Msk
+#  define STM32_SPI_CPOL_Msk            SPI_CR1_CPOL_Msk
+#endif
+#ifdef SPI_CFG2_CPOL_Msk
+#  define STM32_SPI_CPOL_Msk            SPI_CFG2_CPOL_Msk
+#endif
+
+/**
+ * @name   Override the SPI mode values
```

Comment:
I am assuming if I do the following it will make CI/CD happy (based on the `HAVE_SPI_CLK_T` below). Is this the right thing to do?
```suggestion
 * @brief   Override the SPI mode values
```

Reply:
you can test if the CI is happy with a change by running the dist/tools/doccheck locally

reading 
https://www.doxygen.nl/manual/grouping.html#memgroup
https://www.doxygen.nl/manual/commands.html#cmdname

i think it would be better to have it like i will write in the suggesting comment

Reply:
https://ci.riot-os.org/results/aa8dea468fc9407eb2b200ad973e7eda/doc-preview/cpu__spi_8h.html

Reply:
Thanks for the links. I setup to build the docs locally and am now playing around with your suggestion. I am seeing some weird behavior. According to those links, member groups may not be nested. The entire file is in a groups. This is a common idiom used through-out the RIOT source tree. This means that anywhere the member group mechanism is used, we are nesting them inside the one that wraps the entire file. Maybe this explains the weird things I am seeing?

Reply:
After doing more research I'm 99% that is what is going on here. The file wide member group is being canceled as soon as Dozygen sees a nested group inside it. then at the end of the nested group, Doxgen is placing all documents outside of the file-wide one. Essentially, Doxygen does not seem to maintain a stack for managing groups. It just enters the group it last sees and when it hits the end of a group it pops out to non-group namespace.

Reply:
I created #20537 to track this issue separately.